ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -351,6 +355,29 @@ tests:
       YQ_VERSION: 4.30.5
     workflow: openshift-e2e-ibmcloud-heterogeneous-upi
   timeout: 6h0m0s
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le-cucushift
+  cron: 20 23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ppc64le and @ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes
+        and not @fips'
+      GO_VERSION: 1.20.7
+      JQ_VERSION: "1.6"
+      LOKI_ENABLED: "false"
+      OCP_VERSION: "4.14"
```

Comment:
Hey Punith, since we have cleanup jobs that look at 4-14 workspaces. We may need to identify this job as 4.14-cucushift and tag the workspace and transit gateway as such. Can you look at that? This way the cleanup from one job doesn't kill another.